src/views/admin/RolleCreationView.vue

```diff
@@ -26,57 +29,82 @@
   const { t }: Composer = useI18n({ useScope: 'global' })
   const router: Router = useRouter()
 
+  const validationSchema: TypedSchema = toTypedSchema(
```

Comment:
@jaggarnaut  Gerade gesehen, dass du schon Vee-Validate integriert hattest in SPSH-399 für RolleCreationView.vue ... Soll Ich das hier rückgängig machen? Dann haben wir wenige Konflikte.

Reply:
Gerne